docs/user/interfaces/protocol.rst

```diff
@@ -355,9 +356,41 @@ SQL query::
 
 Result set::
 
+	firstname|lastname|age
 	Nanette|Bates|28
 	Amber|Duke|32
 	Dale|Adams|33
 	Hattie|Bond|36
-	
 
+
+The formatter sanitizes the raw result with the following rules:
+
+1. If there exists one or more pipes ('|'), the sanitizer will quote the cell with double quotes.
+
+For example::
+
+    >> curl -H 'Content-Type: application/json' -X PUT localhost:9200/userdata/_doc/1?refresh=true -d '{
+      "+firstname": "-Hattie",
+      "=lastname": "@Bond",
+      "address": "671 Bristol Street|, Dente, TN"
+    }'
+	>> curl -H 'Content-Type: application/json' -X POST localhost:9200/_opendistro/_sql?format=csv -d '{
+	  "query" : "SELECT firstname, lastname, address FROM userdata"
+	}'
+
+Result set::
+
+    '+firstname|'=lastname|address
+    'Hattie|'@Bond|"671 Bristol Street|, Dente, TN"
```

Comment:
Since we don't do sanitize for RAW, the 2nd column is supposed to be `@Bond` rather than `'@Bond`?

And the rest of the doc should be removed?

Reply:
Done, thanks for quality control!